src/pages/PublicProfile.vue

```diff
@@ -240,7 +265,94 @@ export default {
             input-type="textarea"
           />
         </section>
+        <QSeparator class="q-mt-xl" />
+        <section
+          v-show="!(selectedUser.id && !isCurrentUser && !selectedUser.metadata)"
+          class="q-px-sm"
+        >
+        <h2 class="text-h4 text-weight-medium stl-content-entry">Social links</h2>
```

Comment:
@Digital4design All copy should be in language definition files https://github.com/justinschuldt/syr-marketplace/tree/dev/src/i18n/source
Please add the copy for this to the `user.yaml` file under a new key named `social_links_label`, then you can update this to use the value with: 
```suggestion
          <AppContent
            v-if="selectedUser.metadata"
            tag="h2"
            class="text-h4 text-weight-medium"
            entry="user"
            field="social_links_label"
          />
```
Note, it is much better to match the existing coding style for showing a label, like in this case to use `<AppContent ... />`, rather than using a plain `<h2>` and adding your own styles.

Reply:
done as per you described